helmfile/charts/init-harbor/values.yaml

```diff
@@ -8,3 +8,4 @@ oidc:
   endpoint: ""
   verifyCert: true
   groupClaimName: groups
+  adminGroupName: adminGroupName
```

Comment:
Is it possible to default to OIDC being disabled? It's not a huge risk, but it would be nice to not assume that you won't accidentally have a group called "groups" or "adminGroupName" which shouldn't get access. :laughing:

Reply:
You're totally correct. I'll change it to `""` instead.